package.json

```diff
@@ -58,6 +55,9 @@
     "styled-system": "5.1.2"
   },
   "devDependencies": {
+    "@testing-library/react": "9.4.0",
+    "@types/styled-components": "^4.4.0",
+    "@types/styled-system": "5.1.2",
```

Comment:
I'm not sure if it's safe to move `@types/styled-system` out of our regular dependencies - I vaguely remember them needing to be included to work with applications using TS. @smockle @BinaryMuse do you remember anything about this? Moving `@types/styled-components` into devDeps should be fine since we recommend folks include that directly in their application anyways.

Reply:
The `@types` namespace is a collection of [TypeScript declaration files](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html) from the [DefinitelyTyped repo](https://github.com/DefinitelyTyped/DefinitelyTyped).<sup>[1]</sup>

TypeScript declaration files should be installed as a dev dependency since they are only used by the TypeScript compiler to give type information about 3rd party packages.

[1]: https://github.com/DefinitelyTyped/DefinitelyTyped#what-exactly-is-the-relationship-between-this-repository-and-the-types-packages-on-npm

Reply:
> @smockle @BinaryMuse do you remember anything about this?

Yup, this PR outlines the issue: https://github.com/primer/components/pull/630, and there was additional discussion in #ui-frameworks: https://github.slack.com/archives/CMZ4DC9BL/p1571768864193200

Reply:
> @smockle @BinaryMuse do you remember anything about this?

Yup, this PR outlines the issue: https://github.com/primer/components/pull/630, and there was additional discussion in Slack.

https://github.com/primer/components/issues/782 and https://github.com/primer/components/pull/785 are somewhat-related, too.

Reply:
Thanks @smockle!

Reply:
Let's definitely move the testing library into devDeps though!

Reply:
I'll move this line out so we can unblock this PR. But I'm interested in this statement in #630:

>By moving styled-components from devDependencies to dependencies, it will be installed alongside @primer/components, compilation will get farther, but it won’t succeed unless @types/styled-components is also installed, so this PR adds that package to dependencies too.

What would be a minimal reproduction case for this issue? I tried reproducing by creating a new project and linking this branch of `@primer/components` and I'm able to compile the app fine.

Reply:
> What would be a minimal reproduction case for this issue?

I created one here: https://github.com/smockle/primer-ts-demo. If you follow the instructions in the README, you should see a build error after step 3 which is resolved by step 4, as demonstrated by step 5.

Reply:
Thanks for the reproduction @smockle ✨

Reply:
@koddsson it looks like we also need to move `@types/styled-components` back into regular dependencies! If you are cool with that I can make the change and merge this into the release branch going out tomorrow 👍